ci/scripts/install_softhsm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+#Install softhsm
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    brew install softhsm
```

Comment:
I'm pretty sure brew installs softhsm 2.6. This version of softhsm hasn't played well with fabric in the past and could also have similar issues with the FSC implementation of HSM support as it's likely to be using similar code. Unfortunately I can't remember what the problem was with softhsm 2.6, but it's explicitly documented in the latest fabric docs not to use 2.6.

Reply:
Interesting,

I was looking for this statement in the docs but cannot find it. https://hyperledger-fabric.readthedocs.io/en/latest/hsm.html

Reply:
@mbrandenburger You will find it here https://hyperledger-fabric.readthedocs.io/en/latest/dev-setup/devenv.html
I'm trying to remember what the conflict was but has been so long since I worked on fabric HSM I can't remember. It's also possible now that newer versions of the Go C Runtime addressed the issue that occurred with Softhsm 2.6
@sapthasurendran This could still be a problem with fabric though even if the Go client application doesn't exhibit a problem, so might be good to test the newer versions of fabric with Softhsm 2.6 again to see if it is resolved

Reply:
@mbrandenburger The integration test as well as fabric-samples seems to work fine with soft-hsm 2.6 version. I think this PR is good to merge. 
@davidkel I will  test Fabric with softhsm 2.6v on a linux vm as you suggested as a side project .